### DISCUSSION TITLE
WYSIWYG table & cell properties not respected on live page

### DISCUSSION THREAD

**Discussion by jamescobalt** [requarks/wiki#3556] [Error / Bug Report]
**Describe the bug**
In the WYSIWYG page editor, table properties _and_ table cell properties for border & background do not carry over to the published content, and neither does table dimensions. 
Disabling Sanitize HTML will allow most custom border and background styles to show, but borders render with "1px solid #cfd8dc" when set to None and table width is still not respected. Adding the "hidden" border property as a border option might work as an alternative to "none", but there's still a shadow being added to the table (see screenshots).

**To Reproduce**
1. Create a page via WYSIWYG editor
2. Insert a multi column table
3. Set the table width to 100% or some other value
4. Keep the table borders at none
5. Apply custom border and background colors to a cell
6. Save the page and preview
7. Disable Sanitize HTML and check again

**Expected behavior**
The published table would look the same as the editor view (or perhaps the editor view would have very subtle cell borders when borders are set to none, and these would vanish on published page).

**Screenshots**
How it appears in the editor:
<img width="1074" alt="Screen Shot 2020-09-20 at 10 06 03 PM" src="https://user-images.githubusercontent.com/8231171/93728329-7a5b8480-fb8d-11ea-9256-9cb080e6f75b.png">

How it appears when published:
<img width="1161" alt="Screen Shot 2020-09-20 at 10 06 32 PM" src="https://user-images.githubusercontent.com/8231171/93728337-8ba49100-fb8d-11ea-8065-460af19da469.png">

Box shadow and border being added to <td> element on live page:
<img width="424" alt="Screen Shot 2020-09-20 at 10 25 39 PM" src="https://user-images.githubusercontent.com/8231171/93728932-38800d80-fb90-11ea-9c5e-9dbb45cff4f6.png">

**Host Info (please complete the following information):**
 - OS: [e.g. Ubuntu 18.04, Docker]
 - Wiki.js version: [e.g. 2.5.144]
 - Database engine: [e.g. postgres 11.8]
 - Client: Chrome 85, OS X

**Additional context**
I like turtles.

**Comment by Maxzor** [comment]
I have a 2.5.144 install.
Setting a table centered does not center it, it centers the cells's content. 
![image](https://user-images.githubusercontent.com/16599683/94622349-521d0580-02b2-11eb-84fd-d769b494a741.png)
Setting table left or right 'disanchor' it, I would expect it to keep being anchored?  
![image](https://user-images.githubusercontent.com/16599683/94622611-cce62080-02b2-11eb-955c-6b24ce18ae91.png)

System :
Ubu20, pg12,node12

Additional context :
Tried disabling the html sanitizer

**Comment by Maxzor** [comment]
When set to 100% width, the table does not always take 100% of the width :
![image](https://user-images.githubusercontent.com/16599683/94624063-eb99e680-02b5-11eb-8633-8627c0f9ef64.png)

**Reply by TGF-CM** [reply]
This is the workaround I am using right now.
- Edit page with intended full width table in WYSIWYG editor
- Convert page to raw html
- Edit raw html to copy the `style=` from `figure`, to `table`

**Comment by mbystryantsev** [comment]
I have similar issue — table `width` property on rendering not affects table itself, it affects only `figure` table container, but table does not fit 100% of container.
![wiki-js-table-width-bug-table-properties](https://user-images.githubusercontent.com/1485514/116125286-f7c01800-a6cd-11eb-9a50-fb9ff3ff7c98.png)
![wiki-js-table-width-bug-rendered-width](https://user-images.githubusercontent.com/1485514/116125297-fa227200-a6cd-11eb-99e4-6d55cf43df8a.png)
![wiki-js-table-width-bug-1](https://user-images.githubusercontent.com/1485514/116125303-fd1d6280-a6cd-11eb-8f7b-6cb81eb85b05.png)
